.github/issue_template/01_bugreport.md

```diff
@@ -0,0 +1,58 @@
+---
+name: 🐞 Bug Report
+about: Report a bug to help us improve MSBuild.
+title: ''
+labels:
```

Comment:
Can we use labels, if people don't usually have permissions to add labels? we have a `bug` label.

Reply:
Though if we start doing that, we'll have to change our triage policy away from "has no label". That's fine!

Reply:
Do we want to create some sort of label for the "my build is broken" template to assign? Maybe we already have one?